src/System.Memory/tests/Performance/System.Memory.Performance.Tests.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <Compile Include="Perf.Span.Clear.cs" />
     <Compile Include="Perf.Span.Fill.cs" />
+    <Compile Include="Perf.Span.IndexOf.cs" />
```

Comment:
I suppose you just forgot in a different PR to add this right?

Reply:
Correct.